i18n/bundle/bundle.go

```diff
@@ -260,6 +261,22 @@ func (b *Bundle) translate(lang *language.Language, translationID string, args .
 			dataMap["Count"] = count
 			data = dataMap
 		}
+	} else {
+		dataMap := toMap(data)
+		if c, ok := dataMap["Count"]; ok {
+			if cStr, ok := c.(string); ok {
+				var err error
+				count, err = strconv.Atoi(cStr)
```

Comment:
Why is this more complicated than the original PR (#49)?

You don't need to convert the string to a number because the library already handles this conversion later on this line:

```go
p, _ := lang.Plural(count)
```